sovrin_client/cli/genesisTxns.py

```diff
@@ -1,10 +1,10 @@
+from plenum.common.txn import STEWARD, TRUST_ANCHOR
 from plenum.common.types import f
 
-from sovrin_common.txn import TXN_TYPE, TARGET_NYM, ROLE, TXN_ID, NYM,\
-    STEWARD, SPONSOR
+from sovrin_common.txn import TXN_TYPE, TARGET_NYM, ROLE, TXN_ID, NYM
 
 STEWARD_SEED = b'steward seed used for signer....'
-SPONSOR_SEED = b'sponsors are people too.........'
+TRUST_ANCHOR_SEED = b'trust anchors are people too....'
```

Comment:
Actually for me it looks like this seed isn't used anywhere.
However, I reverted the change for backward-compatibility